libraries/rush-lib/src/cli/actions/AddAction.ts

```diff
@@ -90,6 +96,12 @@ export class AddAction extends BaseAddAndRemoveAction {
       );
     }
 
+    if (this._devDependencyFlag.value && this._peerDependencyFlag.value) {
+      throw new Error(
```

Comment:
Usually you add a dependency as both a `peerDependency` and as a `devDependency` so it's available during development. Seems natural to support both flags.

Reply:
Now that you say it, that does make sense... either flag turns off "regular" dependency, but you can specify both.

Reply:
There is a small case that peer and dev dependency often would have different version specified - e.g. peer is `^17 || ^18`, but dev is `~18.0.3`, but in the context of monorepo I'd take the convenience of adding in one command.